### DISCUSSION TITLE
Sql server

### DISCUSSION THREAD

**Discussion by MostafaGarhy1** [medz/dart-orm#409] [Q&A] [answered]
Can use this package with SQL server ?

**Comment by medz** [comment, marked as answer]
@MostafaGarhy1 Yes

**Reply by MostafaGarhy1** [reply]
Thanks for your response and I have question 
Can I pass the IP and database name to connect ?

**Reply by medz** [reply]
@MostafaGarhy1 https://www.prisma.io/docs/orm/reference/connection-urls#microsoft-sql-server 

Here is the documentation on SQL Server connection string.

```prisma
datasource db {
     provider = "sqlserver"
     url = "sqlserver://localhost:1433;initial catalog=sample;user=sa;password=mypassword;"
}
```

Hope it helps.

**Reply by MostafaGarhy1** [reply]
Thanks for help
